packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -231,16 +230,19 @@ export class SchemaConverter implements SimpleDbSchemaConverter {
     const documentsStore = txn.store<DbRemoteDocumentKey, DbRemoteDocument>(
       DbRemoteDocument.store
     );
+    const globalTargetStore = txn.store<DbTargetGlobalKey, DbTargetGlobal>(
+      DbTargetGlobal.store
+    );
 
-    return getHighestListenSequenceNumber(txn).next(currentSequenceNumber => {
+    return globalTargetStore.get(DbTargetGlobal.key).next(metadata => {
       const writeSentinelKey = (
         path: ResourcePath
       ): PersistencePromise<void> => {
         return documentTargetStore.put(
           new DbTargetDocument(
             0,
             encodeResourcePath(path),
-            currentSequenceNumber
+            metadata!.highestListenSequenceNumber!
```

Comment:
We set the metadata earlier in the migration. It should exist at this point.

Reply:
I'm wary of this style of commenting (review-only). The reviewer of this PR isn't the only one who's ever going to read this code.

Don't get me wrong--there's definitely a place for this kind of thing, but this comment in particular seems reasonable to just make in the code itself.

Reply:
Added a debug assert that contains this message.